Source/Extensions/NSURLQueryItem.swift

```diff
@@ -0,0 +1,41 @@
+public extension NSURLQueryItem {
+  static func initAndEncode(key: String, value: String) -> NSURLQueryItem? {
```

Comment:
It seems weird that we need this. Does `NSURLQueryItem(name: value:)` not do this automatically?